ZAPD/Main.cpp

```diff
@@ -115,7 +115,7 @@ int main(int argc, char* argv[])
 	}
 
 	// Parse other "commands"
-	for (int i = 2; i < argc; i++)
+	for (int32_t i = 2; i < argc; i++)
```

Comment:
```suggestion
	for (size_t i = 2; i < argc; i++)
```

Reply:
Since `argc` is an `int` doesnt it make sense for it to stay an `int` here?

Reply:
Yeah, you are right. I didn't realize it was using `argc`.